Tests/App/config/config_test.yaml

```diff
@@ -0,0 +1,23 @@
+parameters:
+    database_path: '%kernel.root_dir%/../../build/test.db'
+
+framework:
+    trusted_hosts: ~
+    secret: "test"
+    test: ~
+
+doctrine:
+    dbal:
+        driver:   pdo_sqlite
+        dbname:   pendingaction_bundle_test
+        user:     root
+        charset:  UTF8
+        path:     '%database_path%'
+    orm:
+        auto_generate_proxy_classes: true
+        auto_mapping: false
+        mappings:
```

Comment:
This part is useless too: doctrine should automatically register bundle entities if they're in the `Entity` bundle directory

Reply:
Doesn't seem to work without this part of the code